chia/full_node/full_node.py

```diff
@@ -128,7 +129,17 @@ def __init__(
         self.uncompact_task = None
         self.compact_vdf_requests: Set[bytes32] = set()
         self.log = logging.getLogger(name if name else __name__)
-
+        self.capabilities = capabilities
+        if config.get("disable_capabilities"):
+            try:
+                disable_capabilities_values = config["disable_capabilities"]
+                if Capability.BASE.value in disable_capabilities_values:
+                    disable_capabilities_values.remove(Capability.BASE.value)
+                self.capabilities = [
```

Comment:
I would have thought this would be a bitfield. is this a list of strings?

Reply:
agree, but it's unfortunately defined like this in Handshake protocol: `capabilities: List[Tuple[uint16, str]]`